dataservice/api/common/custom_fields.py

```diff
@@ -95,3 +96,19 @@ def _deserialize(self, value, attr, data):
             raise ValidationError('Invalid date or datetime string')
 
         return value
+
+
+class EnumColumn(fields.Field):
+    """Validates against a given set of enumerated values."""
+
+    def __init__(self, enum, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.enum = enum
+        self.validators.insert(0,
```

Comment:
Does the order matter here? Maybe more clean to use `append()` instead, so no one is confused by the extra argument.